notebooks/238-deeployd-if/deep-floyd-if.ipynb

```diff
@@ -0,0 +1,783 @@
+{
```

Comment:
<p>I would clarify more that company asks your explicit acceptance of license and uses HF account for that. BTW, this also could be useful to specify in the disclaimer in the beginning of notebook. Person should be aware that this will happen before starting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1039/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Mentioned it in the NOTE section.</p>